pbf/views.py

```diff
@@ -807,6 +822,19 @@ def gain_energy(request, player_id):
     compute_card_thresholds(player)
     return with_log_trigger(render(request, 'energy.html', {'player': player}))
 
+def change_spirit_specific_resource(request, player_id, amount):
+    amount = int(amount)
+    player = get_object_or_404(GamePlayer, pk=player_id)
+    player.spirit_specific_resource += amount
+    player.save()
+
+    # As of this writing, no resource affects card thresholds.
+    # compute_card_thresholds(player)
+
+    # The spirit-specific resource is displayed in energy.html,
+    # because some of them can change simultaneously with energy (e.g. Rot).
```

Comment:
Looks like you had a PR up for this but then closed it, presumably to deal with merge conflicts? Again, so sorry for my delay and thank you so much for these 🙏 

In short: I totally agree with the concept of spirit-specific resources.